superset-frontend/src/components/Button/index.tsx

```diff
@@ -230,7 +231,11 @@ export default function Button(props: ButtonProps) {
       >
         {/* this ternary wraps the button in a span so that the tooltip shows up
         when the button is disabled.  */}
-        {disabled ? <span>{button}</span> : button}
+        {disabled ? (
+          <span style={{ cursor: 'not-allowed' }}>{button}</span>
```

Comment:
I know this is a tiny instance of a `style` prop, but it would be preferable to use the `css` prop here. See the [Emotion styling guidelines](https://github.com/apache/superset/wiki/Emotion-Styling-Guidelines-and-Best-Practices) for more info. 

Eventually we want to get rid of all of the `style` props in favor of `css` so it's easy to expand and theme these things, and create a nice library of these shared styles eventually.